plugins/reporters/static-html/src/funTest/assets/static-html-reporter-test-expected-output.html

```diff
@@ -1071,7 +1067,7 @@ <h3 class="">Packages</h3>
                   </td>
                   <td><em>Concluded License:</em><dl>
                         <dd>
-                           <div><a href="https://archive.softwareheritage.org/browse/content/sha1_git:a612ad9813b006ce81d1ee438dd784da99a54007">MPL-2.0</a> OR <a href="https://archive.softwareheritage.org/browse/content/sha1_git:2aa9b989e0fec475baf21544f8613cb53d03c3f7">EPL-1.0</a></div>
+                           <div><a href="https://archive.softwareheritage.org/browse/content/sha1_git:2aa9b989e0fec475baf21544f8613cb53d03c3f7">EPL-1.0</a> OR <a href="https://archive.softwareheritage.org/browse/content/sha1_git:a612ad9813b006ce81d1ee438dd784da99a54007">MPL-2.0</a></div>
```

Comment:
From just quickly looking at this: Shouldn't things just get reordered? But it seems that original content got lost.

Reply:
I hope I'm not missing anything, but I believe nothing got lost but really just re-ordered.
Is this maybe a GitHub UI rendering issue in the diff view?

before:
```
<div><a href="https://archive.softwareheritage.org/browse/content/sha1_git:a612ad9813b006ce81d1ee438dd784da99a54007">MPL-2.0</a> OR <a href="https://archive.softwareheritage.org/browse/content/sha1_git:2aa9b989e0fec475baf21544f8613cb53d03c3f7">EPL-1.0</a></div>
```
after:
```
<div><a href="https://archive.softwareheritage.org/browse/content/sha1_git:2aa9b989e0fec475baf21544f8613cb53d03c3f7">EPL-1.0</a> OR <a href="https://archive.softwareheritage.org/browse/content/sha1_git:a612ad9813b006ce81d1ee438dd784da99a54007">MPL-2.0</a></div>
```

Reply:
Ah, I got confused because I was expecting lines to get reordered. Instead, terms with a single line got reordered 👍🏻